src/design-tokens/sizes.stories.mdx

```diff
@@ -0,0 +1,44 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import * as msTokens from './modular-scale.yml';
+import * as sizeTokens from './sizes.yml';
+let msStepRows = [];
+for (let i = -4; i <= 10; i++) {
```

Comment:
We could also potentially make these -4 and 10 design tokens, though I don't know that we need to

Reply:
Why?

I'm not trying to be argumentative, I'm just trying to understand the use case.

Reply:
Thinking about this more I don't think there's a very compelling use case.

I was thinking it could help to document the range of modular scale values devs should use. However, it already displays these values in the docs, and these numbers don't affect the SCSS, so I don't see a great reason to make these design tokens.